### DISCUSSION TITLE
Supply Chain Shortage Workaround: STELLA-1 display screen.

### DISCUSSION THREAD

**Discussion by paulmirel** [STELLA-Landsat/STELLA#4] [Build Notes: How do I make a STELLA?]
The 2.4" TFT touchscreen display, the Featherwing version (https://www.adafruit.com/product/3315) is out of stock. 

We can use the non-Featherwing version, https://www.adafruit.com/product/2478, instead. 

This substitution of the freestanding display for the Featherwing version is a bit cumbersome. I plan to publish instructions for this work soon. 

The shopping list for the substitution is: 
Display: https://www.adafruit.com/product/2478
Touch Controller: https://www.adafruit.com/product/5423
Short Headers: https://www.adafruit.com/product/2940 (These are also out of stock, but you can cut these ones to length: https://www.adafruit.com/product/5585) 
Mini Protoboard: https://www.sparkfun.com/products/12702

Instructions: Satisfied 2023-02-28 PM

P

**Comment by paulmirel** [comment]
Instructions for building the alternative display for STELLA-1 are available here: https://landsat.gsfc.nasa.gov/stella/stella-one/stella-one-build-instructions/#altAssembly


We've updated the STELLA-1 software to version 6.2.0, which includes support for the alternative display. Software automatically switches to whichever display is present: the original TFT Featherwing display, the alt TFT display, or no display at all, which puts the software in drone mode.

**Comment by paulmirel** [comment]
We are finding that the screen press function is less responsive than it was with the Featherwing display. You have to push a bit harder, and hold the push for a bit longer, to get the three screen buttons to respond.

**Reply by ArsenioDev** [reply]
This may be due to a different touch layer or graphics pipeline slowing down the execution loop time
